tests/weekly_tests/vivado.yaml

```diff
@@ -1,4 +1,4 @@
-design_dirs:
+designs:
```

Comment:
Are you sure this works?  I thought you needed `design_dirs` when you want it to iterate through all subdirectories.

Reply:
I thought it worked because the test actually runs unlike when it's `design_dirs` but I didn't notice it wasn't going into every design in each subdirectory. The reason I made this change in the first place because the test won't run with it as `design_dirs` because of an issue with how `design_dirs` works.

Reply:
Actually I think I misspoke before, I think using `designs` works fine. I made a change that made the test run using `design_dirs` and it did the same thing as when it was `design`

Reply:
OK, didn't realize it worked.  I've opened #418